Source/bmalloc/bmalloc/AllocationCounts.h

```diff
@@ -110,3 +110,10 @@
 #define BENABLE_PROFILE_COMPACTIBLE_TRY_ALLOCATION 0
 #define BPROFILE_TRY_ALLOCATION_COMPACTIBLE(ptr, size) do { } while (false)
 #endif
+
+#ifdef BPROFILE_ZERO_FILL_PAGE
+#define BENABLE_PROFILE_ZERO_FILL_PAGE 1
+#else
+#define BENABLE_PROFILE_ZERO_FILL_PAGE 0
+#define BPROFILE_ZERO_FILL_PAGE(ptr, size, flags, tag) (true)
```

Comment:
Let's make this an empty `do { } while (false)` block like the other macros, seems like that'd address the build issues on EWS?